client/tooltool.py

```diff
@@ -703,6 +708,216 @@ def touch(f):
         log.warn('impossible to update utime of file %s' % f)
 
 
+def retrier(attempts=5, sleeptime=10, max_sleeptime=300, sleepscale=1.5, jitter=1):
```

Comment:
please add a comment to explain that those are from the redo project

Reply:
sorry, not done yet. ran tox and got a false positive looks like.